build.gradle

```diff
@@ -39,15 +39,6 @@ dependencies {
 	testImplementation 'io.projectreactor:reactor-test'
 	implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.12.0'
 
-//	testImplementation('com.codeborne:selenide:5.25.0')
-//	testImplementation("io.qameta.allure:allure-selenide:${allureVersion}")
-//	testImplementation('org.assertj:assertj-core:3.21.0')
-//	testImplementation('com.google.truth:truth:1.1.3')
-//	testImplementation('org.hamcrest:hamcrest:2.2')
-//	testImplementation('org.slf4j:slf4j-simple:1.7.30')
-//
-//	testImplementation("org.junit.jupiter:junit-jupiter-api:${junitVersion}")
-//	testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:${junitVersion}")
 }
```

Comment:
Has there been a change in performance since removing dependencies?

Reply:
i think it might build faster now, there's also the fact that we are not using selenide anymore so we don't need them.